Source/model/scene/Group.swift

```diff
@@ -31,6 +31,7 @@ open class Group: Node {
             tag: tag
         )
 
+        self.contents = contents
```

Comment:
You change both `contentsVar` and `contents` which is not necessary, because `contentsVar` will be updated automatically.

Reply:
`node` property for `contentsVar` is being set up here, not the contents which indeed will be updated automatically